generators/server/templates/gradle/profile_dev.gradle.ejs

```diff
@@ -45,7 +45,11 @@ bootRun {
 
 <%_ if (!skipClient) { _%>
 task webpackBuildDev(type: <%= _.upperFirst(clientPackageManager) %>Task) {
+    <%_ if (clientPackageManager==='npm') { _%>
```

Comment:
For readability I would but spaces here: `clientPackageManager === 'npm'` (and at all other places too).

Reply:
:+1: Done